test/e2e-common.sh

```diff
@@ -128,6 +128,13 @@ function parse_flags() {
       readonly INGRESS_CLASS="contour.ingress.networking.knative.dev"
       return 2
       ;;
+    --kong-version)
+      # currently, the value of --kong-version is ignored
+      # latest version of Kong pinned in third_party will be installed
```

Comment:
Question: It's interesting that those flags for ingress versions are implemented but ignored. Do we plan to use them at all?

Reply:
Currently only `Istio` uses the `--istio-version` flag.

https://github.com/knative/serving/blob/db81d221ab37b8bce4b56943d4a80f9224b087b1/test/e2e-common.sh#L63-L67

But other ingresses including Kong do not have the plan for now. IMHO, we should implement it and test multiple ingress versions if many users requested it.